packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/main_executable/TestMainExecutable.py

```diff
@@ -0,0 +1,86 @@
+# TestMainExecutable.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test `@_implementationOnly import` in the main executable
+"""
+import commands
+import lldb
+from lldbsuite.test.lldbtest import *
+import lldbsuite.test.decorators as decorators
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import os.path
+import time
+import unittest2
+
+class TestMainExecutable(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+
+    def launch_and_break(self):
```

Comment:
This entire function can be replaced by `lldbutil.run_to_source_breakpoint(self, "break here", SBFileSpec("main.swift"), exe_name=self.getBuildArtifact("main"))`

Reply:
Ah! Where would I learn about this? Or should I just take a look at everything in lldbutil?

Reply:
I don't think it's documented anywhere, but arguably it should.

Reply:
The example test uses it: https://github.com/apple/swift-lldb/blob/stable/packages/Python/lldbsuite/test/sample_test/TestSampleTest.py

I know.. "We have an example test?"